src/main/menu_controls.cpp

```diff
@@ -1068,19 +1068,19 @@ int menuControls_Mouse_Render(bool mouse, bool render)
                     SuperPrint(name, 3, sX+32+(width-32)/4, start_y+(b_base+2*i - scroll_start)*line);
 
                     if(MenuCursor == i && !s_secondaryInput && g_pollingInput)
-                        SuperPrint("...", 3, sX+32, start_y+(b_base+2*i+1 - scroll_start)*line);
+                        SuperPrint("...", 3, sX+32, start_y+(b_base+2*i + 1 - scroll_start)*line);
```

Comment:
`b_base + (2 * i) + 1`

Reply:
Did a full refactor of this section, thanks a lot for this note.